### ISSUE TITLE
About page needs a little improvement.

### ISSUE THREAD

**Issue by salted0dreams** [ZeroOctave/resource-gallery#12]
The teams section needs a little improvement.
Also, contributors section can be added in the page.

**Comment by salted0dreams** [comment]
If it is worth consideration then please assign me this.

**Comment by Astrodevil** [comment]
Go Ahead

**Comment by salted0dreams** [comment]
Hey, I am having trouble setting up the project. Can you help me a little bit? @Astrodevil

**Comment by Astrodevil** [comment]
what's the problem? If you are having problem with cloning repo. Please clone from updated link from README

**Comment by salted0dreams** [comment]
It is resolved now. When i ran 'pnpm preview' , it was showing on the web page. But when i ran did the setup using npm it worked

**Comment by Astrodevil** [comment]
Any update

**Comment by salted0dreams** [comment]
Sorry, I won't be able to do this issue.

**Comment by mfatima05** [comment]
please assign this issue to me

**Comment by Astrodevil** [comment]
Go ahead @mfatima05

**Comment by mfatima05** [comment]
okay thank you sir.......where is the about page code ?

**Comment by Astrodevil** [comment]
Have you checked our project properly before commenting? Are you familiar with stack of this repo? as your gitHub profile looks weird!

About page is in src directory.

**Comment by mfatima05** [comment]
yes I checked the project I was unable to find it and I m just a beginner so that's why my gitHub looks like this

**Comment by mfatima05** [comment]
yes I can work on this

**Comment by Bhuvanchand26** [comment]
Hey @Astrodevil, I can work on this issue and I actually have an idea of how to do this.
please assign me this issue under GSSOC'23.
THANK YOU

**Comment by akanksha984** [comment]
I can work on this issue @Astrodevil
